node/_fs/_fs_chmod.ts

```diff
@@ -12,7 +12,18 @@ export function chmod(
   callback: CallbackWithError,
 ) {
   path = getValidatedPath(path).toString();
-  mode = parseFileMode(mode, "mode");
+
+  try {
+    mode = parseFileMode(mode, "mode");
+  } catch (error) {
+    // TODO(PolarETech): Errors should not be ignored when Deno.chmod is supported on Windows.
+    // https://github.com/denoland/deno_std/issues/2916
+    if (Deno.build.os === "windows") {
```

Comment:
nit: How about branching by this condition before doing try block?

Reply:
Then how about the following?

```ts
  path = getValidatedPath(path).toString();

  if (Deno.build.os === "windows" && mode === null) {
    mode = 0;
  }
  mode = parseFileMode(mode, "mode");

  Deno.chmod(pathModule.toNamespacedPath(path), mode).catch((error) => {
```

I realized that we might need to consider the case where `Deno.chmod` supports Windows in the future, and I assume that if `Deno.chmod` works, then `parseFileMode` must be run.
It could be the case that `Deno.chmod` is supported on Windows but the code here is left as is.
To prepare for such a case, I thought `parseFileMode` would run, and if an error occurred, setting `mode` to a dummy value (0 would be safest) would solve the current issue.

However, the concern remains that if a situation arises in the future where `Deno.chmod` supports Windows but `Deno.FileInfo.mode` does not, the permissions will be set to 0 unexpectedly.

Reply:
Thanks for the explanation. Now the current code just makes sense to me.